### PR TITLE
simple elastic search

### PR DESCRIPTION
**Pull request by igorkasyanchuk**

### CODE REVIEW

File: config/config/chewy.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+test:
```
Comment (romaslmd) [inline config/config/chewy.yml]: curious, why do we need this file?

**Comment by romaslmd** [comment]
code coverage!

File: config/config/chewy.yml (hunk 1 above)
Comment (igorkasyanchuk) [inline config/config/chewy.yml]: oh yes :) wrong folder

**Comment by igorkasyanchuk** [comment]
@romaslmd specs added

but on travis some old specs not works. but locally works

ready to review

**Pull request merged**